source/index.rst

```diff
@@ -1,41 +1,43 @@
-.. PySCF documentation master file, created by
-   sphinx-quickstart on Thu Jan 15 01:55:04 2015.
+
+.. PySCF index file, created by `ablog start` on Mon Feb  8 20:10:55 2021.
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
-Welcome to PySCF documentation!
-===============================
- 
-PySCF is a collection of electronic structure programs powered by Python.
-The package aims to provide a simple, light-weight, and efficient platform for
-quantum chemistry calculations and code development.  The program is developed
-with the following principles:
-
-* Easy to install, to use, to extend and to be embedded;
+Welcome to the PySCF Website!
```

Comment:
But this is the docs, not the website?

Reply:
I think it is part of the website? But the doc part of it?

Reply:
I don't think there's a distinction between the two in the new format, especially once we change the base URL over to pyscf.org

Reply:
That being said it wasn't an intentional change and we can revert it to documentation if people feel strongly